lib/Controller/PicoController.php

```diff
@@ -82,17 +90,21 @@ public function getRoot($site) {
 	 * @param string $site
 	 * @param $page
 	 *
-	 * @return DataDisplayResponse|string
+	 * @return DataDisplayResponse|DataDownloadResponse|string
```

Comment:
I'm not really happy with the bunch of possible return values here but don't have the time to refactor this all properly as well.

@daita Your call if you want to merge as-is or if you want to change this before. :)